### PR TITLE
Update Paredes-Valles NeurIPS 2021

### PR DESCRIPTION
**Pull request by shiba24**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -638,9 +638,9 @@ European Conf. Computer Vision (ECCV), 2020. [Suppl. Mat.](https://www.ecva.net/
 Sensors 2021, 21, 1150. [PDF](https://www.mdpi.com/1424-8220/21/4/1150/pdf)
 - [Paredes-Valles et al., CVPR 2021](#ParedesValles21cvpr),  
 *Back to Event Basics: Self-Supervised Learning of Image Reconstruction for Event Cameras via Photometric Constancy.*
-- <a name="ParedesValles21arxiv"></a>Paredes-Valles, F., Hagenaars, J. J., de Croon, G. C. H. E.,  
-*[Self-Supervised Learning of Event-Based Optical Flow with Spiking Neural Networks](https://arxiv.org/abs/2106.01862)*,  
-arXiv:2106.01862, 2021.
+- <a name="ParedesValles21nips"></a>Paredes-Valles, F., Hagenaars, J. J., de Croon, G. C. H. E.,
```
Comment (guillermogb) [inline README.md:641]: nips -> neurips

File: README.md [hunk 2]
```diff
@@ -1748,7 +1748,7 @@ Int. Symp. Low Power Electronics and Design (ISLPED), 2020. [PDF](https://www.re
 *Learning Monocular Dense Depth from Events.*
 - [Gehrig et al., RA-L, 2021](#Gehrig21ral)   
 *Combining Events and Frames Using Recurrent Asynchronous Multimodal Networks for Monocular Depth Prediction*
-- [Paredes-Valles et al., arXiv 2021](#ParedesValles21arxiv),  
+- [Paredes-Valles et al., arXiv 2021](#ParedesValles21nips),
```
Comment (guillermogb) [inline README.md:1751]: same comment as above.

File: README.md (hunk 1 above)
Comment (guillermogb) [reply README.md:641]: Could you swap the order of the authors, to be consistent with the paper? thanks.

**Review by guillermogb — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (shiba24) [reply README.md]: fixed!

File: README.md [hunk 3]
```diff
@@ -1748,7 +1748,7 @@ Int. Symp. Low Power Electronics and Design (ISLPED), 2020. [PDF](https://www.re
 *Learning Monocular Dense Depth from Events.*
 - [Gehrig et al., RA-L, 2021](#Gehrig21ral)   
 *Combining Events and Frames Using Recurrent Asynchronous Multimodal Networks for Monocular Depth Prediction*
-- [Paredes-Valles et al., arXiv 2021](#ParedesValles21arxiv),  
+- [Hagenaars et al., arXiv 2021](#Hagenaars21neurips),
```
Comment (guillermogb) [inline README.md:1751]: arXiv -> NeurIPS

File: README.md [hunk 4]
```diff
@@ -638,9 +638,9 @@ European Conf. Computer Vision (ECCV), 2020. [Suppl. Mat.](https://www.ecva.net/
 Sensors 2021, 21, 1150. [PDF](https://www.mdpi.com/1424-8220/21/4/1150/pdf)
 - [Paredes-Valles et al., CVPR 2021](#ParedesValles21cvpr),  
 *Back to Event Basics: Self-Supervised Learning of Image Reconstruction for Event Cameras via Photometric Constancy.*
-- <a name="ParedesValles21arxiv"></a>Paredes-Valles, F., Hagenaars, J. J., de Croon, G. C. H. E.,  
-*[Self-Supervised Learning of Event-Based Optical Flow with Spiking Neural Networks](https://arxiv.org/abs/2106.01862)*,  
-arXiv:2106.01862, 2021.
+- <a name="Hagenaars21neurips"></a>Hagenaars, J. J., Paredes-Valles, F., de Croon, G. C. H. E.,
```
Comment (guillermogb) [inline README.md:641]: J. J. -> J.

**Review by guillermogb — CHANGES_REQUESTED** [review]

**Review by guillermogb — APPROVED** [review]

**Comment by likuui** [comment]
来信收到，谢谢！

**Pull request merged**